numpy/lib/utils.py

```diff
@@ -13,9 +13,86 @@
 __all__ = [
     'issubclass_', 'issubsctype', 'issubdtype', 'deprecate',
     'deprecate_with_doc', 'get_include', 'info', 'source', 'who',
-    'lookfor', 'byte_bounds', 'safe_eval'
+    'lookfor', 'byte_bounds', 'safe_eval', 'show_runtime'
     ]
 
+
+def show_runtime():
+    """
+    Print information about various resources in the system
+    including available intrinsic support and BLAS/LAPACK library
+    in use
+    See Also
+    --------
+    show_config : Show libraries in the system on which
+                  NumPy was built.
+    Notes
+    -----
+    1. Information is derived with the help of `threadpoolctl`
+       library.
+    2. In case `threadpoolctl` is not installed, a message is
+       displayed with help on how to install it.
+    3. SIMD related information is derived from `__cpu_features__`,
+       `__cpu_baseline__` and `__cpu_dispatch__`
+    Examples
```

Comment:
Blank line needed
```suggestion
       `__cpu_baseline__` and `__cpu_dispatch__`

    Examples
```

Reply:
Added in [fe63699](https://github.com/numpy/numpy/pull/21468/commits/fe6369961f4ff136161b0dac2ae2ec620dbce5fb)